scripts/lib/CIME/get_timing.py

```diff
@@ -93,6 +93,14 @@ def gettime(self, heading_padded):
         return (0, 0, False)
 
     def getTiming(self):
+        ninst = self.case.get_value("NINST_CPL")
```

Comment:
This variable should be cpl_ninst  - not ninst (since the meaning of ninst really implies the number of instances of prognostic components for a single coupler).  This applies to all the ninst variables in the routine below. We should pick a standard variable name for the number of coupler instances - either ninst_cpl or cpl_ninst and be consistent in its usage. I would prefer ninst_cpl since this is what is used for the xml variable.